go.mod

```diff
@@ -22,7 +22,7 @@ require (
 	github.com/nhatthm/aferocopy v1.0.2
 	github.com/otiai10/copy v1.6.0
 	github.com/pmezard/go-difflib v1.0.0
-	github.com/qri-io/jsonschema v0.2.1
+	github.com/santhosh-tekuri/jsonschema/v5 v5.0.0
```

Comment:
Tak ani v tejto druhej kniznici sa to nakoniec nedalo spravit.
V tejto tretej to uz mam spravene, v tomto PR ju iba vymenim.